scripts/split-e2e-tests.ts

```diff
@@ -379,7 +381,11 @@ const main = (): void => {
         identifier: 'run_e2e_tests',
         buildspec: 'codebuild_specs/run_e2e_tests.yml',
         env: {
-          'compute-type': 'BUILD_GENERAL1_MEDIUM',
+          'compute-type': 'BUILD_GENERAL1_LARGE',
+          variables: {
+            // BUILD_GENERAL1_LARGE has 15GB of memory. 14848MB = 14.5GB. Leave 0.5GB for the OS and other processes.
```

Comment:
Is 0.5GB enough memory for everything else? I assume you're basing this on some heuristic, manual testing, etc?

Reply:
NPM docs suggest leaving 0.5GB https://nodejs.org/api/cli.html#--max-old-space-sizesize-in-megabytes